tex/gregoriotex-spaces.tex

```diff
@@ -792,18 +749,19 @@
 %This function sets the true raises of the two lines above the initial (it has to be called just as the boxes are placed in order to make sure that the values are all correct)
 \def\gre@calculate@annotationtrueraise{%
   \gre@debugmsg{annotation}{Calculating the raise.}%
-  \global\advance\gre@dimen@annotationtrueraise by \gre@dimen@staffheight\relax%
   \gre@debugmsg{annnotation}{Added staff height.}%
-  \global\advance\gre@dimen@annotationtrueraise by \gre@dimen@spacebeneathtext\relax%
   \gre@debugmsg{annnotation}{Added space beneath text.}%
-  \global\advance\gre@dimen@annotationtrueraise by \gre@dimen@currenttranslationheight\relax%
   \gre@debugmsg{annnotation}{Added translation height.}%
-  \global\advance\gre@dimen@annotationtrueraise by \gre@dimen@spacelinestext\relax%
   \gre@debugmsg{annnotation}{Added spacelinestext.}%
```

Comment:
I'd get rid of/consolidate these debug messages now.  I needed them when writing this macro to debug a problem that was occurring in the math, but they're no longer necessary.  Having separate messages also tends to obfuscate the fact that all these calculations are now done in one step.